x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/_legend.scss

```diff
@@ -15,25 +15,25 @@
   height: $euiSizeM;
   width: $euiSizeM;
   background-color: $euiColorGhost;
-  border: 1px solid $euiColorVis1;
+  border: 2px solid $euiColorVis1;
   display: 'inline-block';
 }
 
 .mlJobMapLegend__analytics {
   height: $euiSizeM;
   width: $euiSizeM;
   background-color: $euiColorGhost;
-  border: 1px solid $euiColorVis0;
-  border-radius: $euiBorderRadius;
+  border: 2px solid $euiColorVis0;
+  border-radius: 50%;
```

Comment:
This could be `$euiBorderRadius` (it equates to 4px)

Reply:
Border radius variable here doesn't give us a true circle.